packages/react/src/components/UIShell/SideNav.tsx

```diff
@@ -65,11 +102,11 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
   //   : t('carbon.sidenav.state.closed');
 
   const className = cx({
+    ...(customClassName !== undefined && { [customClassName]: true }),
```

Comment:
What's the reason for this change? The previous `[customClassName]: !!customClassName` is a bit more readable to me